# Single obstacle information
geometry_msgs/Point position  # Global coordinates (x, y, z)
float64 size                  # Approximate size/radius of obstacle
float64 distance              # Distance from robot to obstacle centroid (meters)
int32 point_count            # Number of points in this obstacle cluster